specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iotdps.json

```diff
@@ -0,0 +1,2619 @@
+          "x-ms-enum": {
+            "name": "IotDpsSku",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "OperationInputs": {
+      "description": "Input values for operation results call.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the Provisioning Service to check.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "NameAvailabilityInfo": {
+      "type": "object",
+      "description": "Description of name availability.",
+      "properties": {
+        "nameAvailable": {
+          "description": "specifies if a name is available or not",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "specifies the reason a name is unavailable",
+          "enum": [
+            "Invalid",
+            "AlreadyExists"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "NameUnavailabilityReason",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "description": "message containing a detailed reason name is unavailable",
+          "type": "string"
+        }
+      }
+    },
+    "TagsResource": {
+      "properties": {
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "A container holding only the Tags for a resource, allowing the user to update the tags on a Provisioning Service instance."
+    },
+    "ProvisioningServiceDescriptionListResult": {
+      "type": "object",
+      "description": "List of provisioning service descriptions.",
+      "properties": {
+        "value": {
+          "description": "List of provisioning service descriptions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProvisioningServiceDescription"
+          }
+        },
+        "nextLink": {
+          "description": "the next link",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "VerificationCodeResponse": {
+      "type": "object",
+      "description": "Description of the response of the verification code.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of certificate.",
+          "readOnly": true
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Request etag."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource identifier."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "verificationCode": {
```

Comment:
Is this a secret? Then please annotate with x-ms-secret. GET. https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review

Reply:
@mrohera Can you confirm that this is a secret?

Reply:
x-ms-secret: true added